lxml-4.6.3 download not available for windows, please update to lxml-4.6.4

I am trying to install codechecker on windows.
lxml 4.6.4 is available here https://pypi.org/project/lxml/4.6.4/#files as cp310 for windows.
But lxml-4.6.3 is not: https://pypi.org/project/lxml/4.6.3/#files
I can install lxml-4.6.4 via pip before installing codechecker.
But when installing codechecker, it uninstalls lxml-4.6.4 and tries to install lxml-4.6.3.
Since lxml-4.6.3 is not available, it tries to compile it and fails with an error.
~~~~
  Attempting uninstall: lxml
    Found existing installation: lxml 4.6.4
    Uninstalling lxml-4.6.4:
      Successfully uninstalled lxml-4.6.4
    Running setup.py install for lxml: started
    Running setup.py install for lxml: finished with status 'error'
~~~~
it is annoying in two ways:
1. I am forced to add the gigantic "visualstudio2017buildtools" to my docker image
2. and it fails to compile lxml.

If `pip install codechecker` would agree in using lxml-4.6.4, that would be solved. 

~~~~
    C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DCYTHON_CLINE_IN_TRACEBACK=0 -Isrc -Isrc\lxml\includes -IC:\Python310\include -IC:\Python310\Include -IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\include -IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt -IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared -IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um -IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt -IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt /Tcsrc\lxml\etree.c /Fobuild\temp.win-amd64-3.10\Release\src\lxml\etree.obj -w
    cl : Command line warning D9025 : overriding '/W3' with '/w'
    etree.c
    c:\users\containeradministrator\appdata\local\temp\pip-install-4q5tm6gi\lxml_f1ebe4e79dba4d26a7bd50cda412e2ab\src\lxml\includes/etree_defs.h(14): fatal error C1083: Cannot open include file: 'libxml/xmlversion.h': No such file or directory
    Compile failed: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2

~~~~

--- a/analyzer/requirements.txt
+++ b/analyzer/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 portalocker==2.2.1
 psutil==5.8.0
 PyYAML==5.4.1

--- a/analyzer/requirements_py/dev/requirements.txt
+++ b/analyzer/requirements_py/dev/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 nose==1.3.7
 pycodestyle==2.7.0
 psutil==5.8.0

--- a/analyzer/requirements_py/osx/requirements.txt
+++ b/analyzer/requirements_py/osx/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 portalocker==2.2.1
 psutil==5.8.0
 scan-build==2.0.19

--- a/web/requirements.txt
+++ b/web/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 sqlalchemy==1.3.23
 alembic==1.5.5
 portalocker==2.2.1

--- a/web/requirements_py/db_pg8000/requirements.txt
+++ b/web/requirements_py/db_pg8000/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 sqlalchemy==1.3.23
 alembic==1.5.5
 pg8000==1.15.2

--- a/web/requirements_py/db_psycopg2/requirements.txt
+++ b/web/requirements_py/db_psycopg2/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 sqlalchemy==1.3.23
 alembic==1.5.5
 psycopg2-binary==2.8.6

--- a/web/requirements_py/dev/requirements.txt
+++ b/web/requirements_py/dev/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 sqlalchemy==1.3.23
 pycodestyle==2.7.0
 alembic==1.5.5

--- a/web/requirements_py/osx/requirements.txt
+++ b/web/requirements_py/osx/requirements.txt
@@ -1,4 +1,4 @@
-lxml==4.6.3
+lxml==4.6.4
 alembic==1.5.5
 portalocker==2.2.1
 psutil==5.8.0

